toolchain/c-toolchain.lisp

```diff
@@ -173,8 +173,9 @@
              (if normalizep (normalize-flags sbcl-home val) val))))
     (unless (featurep :sb-linkable-runtime)
       (setf *linkkit-start* nil *linkkit-end* nil))
+    #+openbsd
+    (setf *cc-flags* (list "-I" "/usr/local/include/"))
     (setf *ld* *cc* ;; !
-          #+openbsd *cc-flags* (list "-I" "/usr/local/include/")
```

Comment:
random note: the thinko fixed here demonstrates nicely why, even though it's possible, multi-entry setf's are a bad idea...

Reply:
Also, I think reader conditionals are the wrong tool here, particularly for CLISP that has portable fasls, but CFFI's fasls are not portable anyway. :-(